src/index.js

```diff
@@ -480,6 +490,89 @@ export default class AuthnWidget {
     document.querySelector('#submit').disabled = disabled;
   }
 
+  async registerIdVerificationRequiredEventHandler() {
+    let data = this.store.getStore();
+
+    if (data.errorDetails !== undefined)
+    {
+      document.getElementById("requiredDescription").style.display = "none";
+      document.getElementById("errorDescription").style.display = "block";
+
+      document.getElementById("errorBlock").style.display = "flex";
+      document.getElementById("errorMessage").innerHTML = this.makeIdVerificationErrorMessage(data.errorDetails);
+    }
+    else
+    {
+      document.getElementById("requiredDescription").style.display = "block";
+      document.getElementById("errorDescription").style.display = "none";
+
+      document.getElementById("errorBlock").style.display = "none";
+    }
+
+    const options = {
+      method: 'GET'
+    }
+    let qrUrlRespone = await fetch(data.qrUrl, options);
+    let qrCode = await qrUrlRespone.text();
+    document.getElementById('qrCode').src = qrCode;
+    document.getElementById('qrCodeBlock').style.display = 'block';
+
+    var verificationCodeToken = data.verificationCode.match(/.{1,4}/g);
+    document.getElementById('verificationCode').innerHTML = verificationCodeToken.join(' ');
+  }
+
+  postIdVerificationRequired() {
+    document.getElementById('copy')
+            .addEventListener('click', this.copyCode);
+    this.pollAction(5000);
+  }
+
+  handleIdVerificationInProgress() {
+    this.pollAction(5000);
+  }
+
+  handleIdVerificationFailed() {
+    let data = this.store.getStore();
+
+    if (data.errorDetails !== undefined)
+    {
+      document.getElementById("errorMessage").innerHTML = this.makeIdVerificationErrorMessage(data.errorDetails);
```

Comment:
Same here for the errors

Reply:
As discussed `{{> errors}}` can't be used for `ID_VERIFICATION_FAILED` due to the extra red triangle when displayed.